webapp/src/components/notebook_preview/code_cell_output_data.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import ReactMarkdown from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+import Ansi from 'ansi-to-react';
+import DOMPurify from 'dompurify';
+import katex from 'katex';
+
+import {joinText} from './utils';
+
+import 'katex/dist/katex.min.css';
+
+type Props = {
+    format: string
+    content: string|string[]
+}
+
+const CodeCellOutputData = ({format, content}: Props) => {
+    const sanitizedContent = DOMPurify.sanitize(joinText(content));
```

Comment:
@jupenur Do you have some time to make sure we are not missing an injection attack here?